doc/releases.md

```diff
@@ -1,44 +1,76 @@
 # Releases
 
+See [the HoloViz blog](https://blog.holoviz.org/tag/hvplot.html) for a visual summary of the major features added in a selection of releases.
+
+## Version 0.8.2
+
+**November 24, 2022**
+
+The 0.8.2 release is a bug fix release. It improves compatibility with HoloViews 1.15.2 that emits now a warning when parsing DataFrame with non-string columns. The HoloViz tools are not yet compatible with the recent Bokeh 3, this hvPlot release pins Panel in a way that hvPlot users getting this version will no longer see Bokeh 3 being installed.  The documentation has seen some improvements too; @MarcSkovMadsen has kept improving the docstrings and we congratulate @sophiamyang for her first contribution that made the landing page nicer! Many thanks to @droumis, @Hoxbro, @maximlt, @philippjfr and @MarcSkovMadsen for contributing!
```

Comment:
I would make this sentence into the top entry in the notes (there must be a PR with this pin, right?):

> The HoloViz tools are not yet compatible with the recent Bokeh 3, this hvPlot release pins Panel in a way that hvPlot users getting this version will no longer see Bokeh 3 being installed.

Maybe something like:

```
- Dependencies
   - Added Panel pin to ensure Bokeh <3 until HoloViz ecosystem updated to support Bokeh 3
      < PR link>   
```

Reply:
Yes that's the packaging entry:

```
- Packaging:
  - Add Panel lower pin to ensure Bokeh 3 is not installed ([#974](https://github.com/holoviz/hvplot/pull/974))
```

I'll move it up.

Reply:
Ok, then I would move this entry to the top and remove the sentence in the overall description.

Reply:
I think we should keep the sentence there as I think users can still end up with an environment that first has Bokeh 3 and then pulls an old version of hvPlot/HoloViews/Panel ... Holoviz not yet supporting Bokeh 3 is at least disclaimed more a bit more explicitly with this sentence.

Reply:
In that case, the sentence should be shorter and clearer. Maybe simply:

> Please note that the HoloViz ecosystem is not yet compatible with Bokeh 3.